.github/CONTRIBUTING.md

```diff
@@ -7,6 +7,14 @@
 Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing the bug.
```

Comment:
Also, this line and the heading before that read as follows:
```markdown
### Submitting a bug report
Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing the bug.
```
Since this document is not limited to bug reports now, we may have to rephrase it. Try something like:
```markdown
### Submitting an issue
Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing your concerns.
```
Feel free to rephrase if you can find better phrases to use.